pkg/sosreport/index.html

```diff
@@ -33,7 +33,8 @@
       <img src="sosreport.png" alt="">
       <p translate="yes">This tool will collect system configuration and diagnostic information from this system for use with diagnosing problems with the system.</p>
       <p translate="yes">The collected information will be stored locally on the system.</p>
-      <button translate="yes" class="pf-c-button pf-m-primary"
+      <p id="switch-instructions" translate="yes">You need to switch to "Administrative Access" in order to create reports.</p>
+      <button id="create-button" translate="yes" class="pf-c-button pf-m-primary"
```

Comment:
I have a gut feeling that both elements should be hidden until superuser is initialized, to avoid flicker on initial page load. WDYT?

Reply:
Yes, good point.  This is how the super user indicator in the shell behaves as well: it is hidden during initialization.

`allowed` is "null" for the whole time that sudo is starting up. If you have two factor auth, this is going to be a couple of seconds while sudo asks repeatedly. We want to fix that eventually and make it fail immediately, but there always is the possibility that sudo is slow.

So I'll change that here and also update the docs in `pkg/superuser`.

Reply:
Done.